src/System.Collections/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Collections:
```

Comment:
This should not be a universal ApiCompatBaseline.txt. It should be specific to just uap.